tests/ttnn/integration_tests/unet/test_ttnn_shallow_unet.py

```diff
@@ -321,87 +321,87 @@ def __init__(self):
         super(UNet, self).__init__()
         # Contracting Path
         self.c1 = nn.Conv2d(3, 16, kernel_size=3, padding=1)
-        self.b1 = nn.BatchNorm2d(16)
+        self.b1 = nn.BatchNorm2d(16, momentum=1)
```

Comment:
why did we need to add momentum here? is it used at all in inference time?

Reply:
no, but the batch norm running var will change and exponentially get larger. This leads to the final output being inf values. To keep it steady, I added this var